### PR TITLE
Add brand guidelines

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: src/brand.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+title: Dart Brand Guidelines
+---
+
+The "Dart" name and logo are trademarks owned by Google.
+These Brand Guidelines describe the appropriate uses of the Dart
+trademarks by members of the developer community who have obtained our
+consent to use the trademarks pursuant to the [Dart Terms of Service](/terms).
+These guidelines will ensure that the Dart trademarks are used in a
+manner that promotes Google's mission to provide a free and open source
```
Comment (kwalrath) [inline src/brand.md]: Seems flutter-specific.

File: src/brand.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+Android in record time, and are not associated with objectionable material,
+as determined by Google.  
+
+Use of the Dart trademarks that is not expressly permitted by these
+guidelines is prohibited absent written permission from Google.
+
+The official Dart logos can be found in the [Dart and Flutter logo
+assets](https://drive.google.com/corp/drive/folders/1KXNtO9My2AMpDOF9A9Y_4aj4_BcgmDDT) Drive folder.
+
+## General Rules That Govern the Use of the Dart Trademarks
```
Comment (kwalrath) [inline src/brand.md]: We use sentence case capitalization for headings:

## General rules that govern the use of the Dart trademarks

File: src/brand.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+provided you do not use the Dart trademarks in a way that
+could confuse people into thinking that your site is an official Google
+site or that Google has sponsored or endorsed your site. In the case of
+websites or personal blogs, this means you should not use the Dart
+trademarks as the primary element on the webpage (e.g., in the masthead
+of the webpage or the title of the blog). In the case of social media
+accounts, this means you should not use the Dart trademarks in the
+background, in your profile image or in your social media username.  
+
+## Specific Rules for Proper Usage of the Dart Trademarks
```
Comment (kwalrath) [inline src/brand.md]: Again, we should use sentence case here.

File: src/brand.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+* Don't incorporate the Dart trademarks into your own product names,
+  service names, trademarks, logos, or company names.   
+* Don't display the Dart trademarks in a manner that is misleading,
+  unfair, defamatory, infringing, libelous, disparaging, obscene or
+  otherwise objectionable to Google.
+* Don't use the Dart trademarks on or in connection with the sale of
+  any non-software goods or services (e.g., merchandise such as clothing,
+  pens, and stickers).
+
+## Community Use Exceptions
```
Comment (kwalrath) [inline src/brand.md]: And here

File: src/brand.md (hunk 1 above)
Comment (kwalrath) [reply src/brand.md]: The other file drops the " for crafting...record time"

Comment (mit-mit) [reply src/brand.md]: Done

**Pull request merged**